dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -379,9 +125,8 @@
   {%- set target_table = model.get('alias', model.get('name')) -%}
 
   {%- set strategy_name = config.get('strategy') -%}
-  {%- set unique_key = config.get('unique_key') -%}
   {%- set file_format = config.get('file_format', 'parquet') -%}
-  {%- set table_type = config.get('table_type', 'iceberg') -%}
```

Comment:
I changed the default from `iceberg` to `hive`, to be consistent with the other defaults in dbt-athena